styles/ImageSlider.module.css

```
@@ -0,0 +1,23 @@
+.prev{
+    width: 30px;
+    margin-right:-30px;
+    cursor: pointer;
+}
+
+.next{
+    width: 30px;
+    margin-left: -30px;
+    cursor: pointer;
+}
+
+.key{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    height: 500px;
+}
+
+.img{
```

No biggie, the images get i little distorted if screen is wider than 1280px. Maybe add:
```suggestion
object-fit: cover; 
```
or
```suggestion
object-fit: contain;
```